tests/ai_eval_components/dataset_gen.py

```diff
@@ -90,10 +94,47 @@ def write_dataset_file(dataset_dir_path, component_name, file_index, dataset):
 
 
 if __name__ == "__main__":
+    # Parse command line arguments
+    src_path = BASE_MODULE_PATH + "dataset_src/"
+    dest_path = BASE_DATASET_PATH
+    try:
```

Comment:
Please extract configuration loading to separate method to keep main more readable, as is in `eval.py`